features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunBorderRouter.h

```diff
@@ -35,6 +35,10 @@ typedef struct ws_br_info {
     uint64_t host_timestamp;
     /** Amount of devices in the network. */
     uint16_t device_count;
+    /** Backbone IPv6 Global Address */
+    uint8_t backbone_global_addr[16];
```

Comment:
Mesh interface Global address should be asked from the Wisun interface not from border router

Backbone global address should be asked from backbone interface which is the ethernet interface

Reply:
Should the network manager directly fetch backbone global address from Ethernet interface ?

Reply:
yes

Reply:
Done